cmd/kubermatic-installer/cmd_deploy.go

```diff
@@ -142,6 +144,8 @@ func DeployCommand(logger *logrus.Logger, versions kubermaticversion.Versions) *
 	cmd.PersistentFlags().BoolVar(&opt.MLAForceMLASecrets, "mla-force-secrets", false, "(UserCluster MLA) force reinstallation of mla-secrets Helm chart")
 	cmd.PersistentFlags().BoolVar(&opt.MLAIncludeIap, "mla-include-iap", false, "(UserCluster MLA) Include Identity-Aware Proxy installation")
 
+	cmd.PersistentFlags().BoolVar(&opt.DeployDefaultAppCatalog, "deploy-default-app-catalog", false, "Deploy the default Application Catalog (EE only)")
```

Comment:
If this is EE only, please define a wrapper in the main_ce/main_ee that returns the persistent flags, with CE returning [] and EE returning this one flag.

Reply:
I have pushed how I understood your idea to [`9b298b2` (#12623)](https://github.com/kubermatic/kubermatic/pull/12623/commits/9b298b221da3010684e9d6c43eaa63394f5dc61e). ptal if that is how you envisioned the code to be structured.

Reply:
Exactly. :)